marimo/_plugins/ui/_core/ui_element.py

```diff
@@ -379,6 +379,10 @@ def _update(self, value: S) -> None:
         if self._on_change is not None:
             self._on_change(self._value)
 
+    def _on_update_completion(self) -> None:
+        """Callback to run after the kernel has processed a value update."""
+        return
```

Comment:
This method acts as a post execution hook; the runtime calls it after setting UI element values.